jdaviz/app.py

```diff
@@ -1121,15 +1126,15 @@ def get_subsets(self, subset_name=None, spectral_only=False,
     def _is_subset_spectral(self, subset_region):
         if isinstance(subset_region, SpectralRegion):
             return True
-        elif isinstance(subset_region, list) and len(subset_region) > 0:
+        elif isinstance(subset_region, list):
```

Comment:
why the change here?  Could there still be a list with no length and what would happen now?